presto-main/src/main/java/io/prestosql/sql/planner/NodePartitioningManager.java

```diff
@@ -221,7 +221,7 @@ private ConnectorBucketNodeMap getConnectorBucketNodeMap(Session session, Partit
 
         ToIntFunction<ConnectorSplit> splitBucketFunction = partitioningProvider.getSplitBucketFunction(
                 partitioningHandle.getTransactionHandle().orElse(null),
-                session.toConnectorSession(),
+                session.toConnectorSession(partitioningHandle.getConnectorId().get()),
```

Comment:
There are many places where `session.toConnectorSession()` is called, and only one is updated.

Would it make sense to have a test ensuring at least the "connector partitioning interface" use-cases is working end to end?

Reply:
The title is confusing, I just test a method `ConnectorNodePartitioningProvider#getSplitBucketFunction`, it cannot fetch the session properties.

Reply:
Could you add a test for that?

Reply:
![Screenshot from 2020-10-26 17-49-51](https://user-images.githubusercontent.com/33892550/97157912-b5b42900-17b3-11eb-9fad-21622c6af910.png)
`NodePartitioningManager#getConnectorBucketNodeMap` has the same problem as `ConnectorNodePartitioningProvider#getSplitBucketFunction`.

Reply:
Well, I saw method `Session#toConnectorSession` has been test in many place in presto test code, do we really need a test for that?   :)

Reply:
Please also fix `NodePartitioningManager#getConnectorBucketNodeMap`

Reply:
Sure!

Reply:
Is it possible `partitioningHandle.getConnectorId().isEmpty()` here?

Reply:
![11](https://user-images.githubusercontent.com/33892550/97283208-83213380-187a-11eb-83df-c1d36afd00d0.png)
![22](https://user-images.githubusercontent.com/33892550/97283267-97fdc700-187a-11eb-8f02-e99d9fcf8bd7.png)
I think it won't be null, for it is checked in the previous code.